### PR TITLE
Feature dog sitting page

### PR DESCRIPTION
**Pull request by xiezihen**
Not yet integrated with profile listings, added temporary code(that I didn't commit) to landing to test and display my component.
alt text

### CODE REVIEW

File: client/src/Components/sitterProfile/aboutMeProfile/AboutMeProfile.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { Alert } from "@material-ui/lab";
+const useStyles = makeStyles({
+  profilePic: {
+    width: "140px",
+    height: "140px",
+    alignContent: "center",
+  },
+});
+
+function AboutMeProfile(props) {
```
Comment (bonnieli) [inline client/src/Components/sitterProfile/aboutMeProfile/AboutMeProfile.js:14]:
```suggestion
function AboutMeProfile({profile}) {
```

that way you won't need to initialize another `const` on line 16

File: client/src/Components/sitterProfile/aboutMeProfile/AboutMeProfile.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.paper {
+  max-width: 600px;
+}
+.MuiPaper-root.MuiPaper-elevation2 {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: 230px 90px 50px repeat(2, 50px) auto;
+  grid-template-areas:
```
Comment (bonnieli) [inline client/src/Components/sitterProfile/aboutMeProfile/AboutMeProfile.css:8]: cool use of grid!

File: client/src/Components/sitterProfile/bookSitter/BookSitter.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+const useStyles = makeStyles({
+  bookSitterPaper: {
+    maxWidth: "400px",
+    textAlign: "center",    
+  },
+  picker: {
+    maxWidth: "130px",
+  },
+  bookButton: {
+    background: "tomato",
```
Comment (bonnieli) [inline client/src/Components/sitterProfile/bookSitter/BookSitter.js:16]: i see we're using a mix of useStyles and .css files. we probably want to stay consistent with these things!

Comment (xiezihen) [reply client/src/Components/sitterProfile/bookSitter/BookSitter.js:16]: yeah i wanted to use css grid but I couldn't find a way to do that in use styles, and if i wanna style mui components i cant put that in the css file. every time i search up using css grid in usestyles muis grid stuff always came up.

**Pull request merged**